deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,39 @@
+# Deploy SparkDotnet REPL to HDI Spark Cluster with Notebook Experience
+
+## Goal
+
+This documentation will elaborate the steps on how to deploy SparkDotnet REPL to HDI Spark Cluster and submit jobs through Jupyter Notebook using SparkDotnet.
+
+## Background
+
+We are currently using [dotnet-try](https://github.com/dotnet/try) as our dotnet REPL. ``` dotnet-try ``` can be used as a jupyter kernel and we implement it through [sparkmagic](https://github.com/jupyter-incubator/sparkmagic) and livy. To enable Jupyter Notebook Experience using SparkDotnet, this will include simple manual steps through [Ambari](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-manage-ambari) and [script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux) in HDI Spark Cluster.
+
+## Step
+
+### 1. Go to Ambari, manually stop Livy Server on head node.
+
+Details to be added.
+
+### 2. Run Script Actions on head node and worker node.
+
+Details to be added.
+
+### 3. Go to Ambari, manually start Livy Server on head node.
+
+Details to be added.
+
+### 4. Ambari -> Spark2 -> CONFIGS, set up Custom spark2-defaults.
+
+Details to be added.
```

Comment:
As you did in Step 1, locate Spark2 under the Services on the left menu. At the top of this page, you'll see SUMMARY and CONFIGS. Select CONFIGS, then Custom spark2-defaults, and finally Add Property...

You will add a total of 4 individual properties. Add them 1 one at a time, all using TEXT Property Type in Single property add mode. Make sure you don’t have any extra spaces before or after any of the keys/values.

Property 1:
Key 1: 
spark.dotnet.shell.command

Value 1: 
/usr/share/dotnet-tools/dotnet-try,kernel-server,--default-kernel,csharp

Property 2:
Key 2: 
spark.dotnet.packages

Value 2: 
["nuget: Microsoft.Spark, 0.5.0", "nuget: Microsoft.Spark.Extensions.Delta, 0.5.0"]

Property 3:
Key 3:
spark.dotnet.version

Value 3:
0.5.0

Property 4:
Key 4:
spark.dotnet.interpreter

Value 4: 
try

After adding those 4 properties, select Save. If you see a warning screen of config recommendations, select PROCEED ANYWAY.